source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,105 @@
+
+Supporting hearing impairment is not something that we can do for the users of Video.js. Instead we must indirectly support these users by supporting captions in their videos. In Video.js we have had support for captions and subtitles for some time, internally they are called `TextTracks`.
+
+Supporting vision impairment is harder, but fully in our control. To support this group of users our player must be accessible to screen readers. A screen reader is an application that reads elements off of the screen to the user (as the name implies). HTML has certain rules that must be followed so that a page can be accessible. We will go over the basics of these rules in the next section. Screen readers are further supported by having description tracks that can be read out by screen readers during video playback (that is another type of `TextTrack`).
+
+See the [resources section at the end of this post](#resources) for a list of screen readers.
+
+
+### How do you make a web application screen reader accessible?
+If you use the native elements for the purposes that they were intended you will be most of the way towards beings accessible. Using the native element is the recomended way to make anything accessible for a screen reader. For instance if you use a `<button>` element you will get the following accessiblity attributes (without them actually being on the button):
+* `tabIndex` which allows users to tab to the button
+* `role="button"` which tells the screen reader that this is a button
+* The space and the enter key will both press the button
+
+In some cases, such as in Video.js, it will not be possible to use the native button element. So you will have to mimic the accessible functionality from the list above and use a `div`. For a `div` that wants to be classified as a button:
+* You have to add the `role="button"` attribute to classify it as a button.
+* You have to add a `tabIndex` which will allow the `div` to be navigated to using only the `tab` key
+* You have to add handling for the space and enter key that press the button
+
+{% pullquote %}
+A list of role attribute values can be found on [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Composite_roles).
+{% endpullquote %}
+
+After adding roles to the controls and content in your webpage, the next thing to look over are `aria` attributes. For instance, we use `aria-live` for our `ProgressBar` slider. This is because the `ProgressBar` is always updating when a video is playing.
```

Comment:
"For instance, we use `aria-valuenow` and `aria-valuetext` for our `ProgressBar` slider, to convey the current position of the slider to screen reader users." (`aria-live` on the `ProgressBar` doesn't work exactly as you describe, and it would be really annoying for screen reader users if the progress bar caused the screen reader to announce the changing time as the video plays!!)